resource/opentelemetry-resource-detector-azure/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) Microsoft Corporation.
```

Comment:
This needs review by legal experts regarding which license should this be.

Reply:
Switched to apache 2